Reset 'm_touchPressed' to false in EventHandler::clear()

Ahmad-S792 opened the pull request:
#### 9a050efadba8c42c2a06a2a6b938be2afe1b4e3a
<pre>
Reset &apos;m_touchPressed&apos; to false in EventHandler::clear()

Reset &apos;m_touchPressed&apos; to false in EventHandler::clear()
<a href="https://bugs.webkit.org/show_bug.cgi?id=250952">https://bugs.webkit.org/show_bug.cgi?id=250952</a>

Reviewed by Aditya Keerthi.

Merge - <a href="https://chromium.googlesource.com/chromium/blink/+/b7791c376d099dbb0ecebf51203af5b856afb7f6">https://chromium.googlesource.com/chromium/blink/+/b7791c376d099dbb0ecebf51203af5b856afb7f6</a>

The member variable &apos;m_touchPressed&apos; in EventHandler is true if there
is at least one active touchpoint. This should be reset to false
in EventHandler::clear(), otherwise mousemove events may be
incorrectly treated.

* Source/WebCore/page/EventHandler.cpp:
(EventHandler::clear): Explicitly set &apos;m_touchPressed&apos; to &apos;false&apos;

Canonical link: <a href="https://commits.webkit.org/259288@main">https://commits.webkit.org/259288@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/f8844ac976f3f0f3eb81ef94556e8fc43d40f150

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/6/builds/104455 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/77/builds/13533 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/43/builds/37360 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/8/builds/113734 "Built successfully") | [✅ 🛠 🧪 win](https://ews-build.webkit.org/#/builders/10/builds/173959 "Built successfully and passed tests") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/11/builds/108377 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/76/builds/14637 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/85/builds/4451 "Built successfully") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/36/builds/96800 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/12/builds/112702 "Built successfully") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/19/builds/110222 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/78/builds/11283 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/3/builds/94336 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/35/builds/38885 "Passed tests") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/9/builds/93148 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/73/builds/25947 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/34/builds/80554 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/81/builds/6892 "Built successfully") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/70/builds/27304 "Passed tests") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/82/builds/7018 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/84/builds/3864 "Passed tests") | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/80/builds/13049 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/62/builds/46862 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/74/builds/6404 "Built successfully and passed tests") | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/79/builds/8812 "Built successfully") | | | 
| | | | | 
<!--EWS-Status-Bubble-End-->

pxlcoder commented on Source/WebCore/page/EventHandler.cpp:
```diff
@@ -424,6 +424,7 @@ void EventHandler::clear()
 #endif
     m_maxMouseMovedDuration = 0;
     m_didStartDrag = false;
+    m_touchPressed = false;
```
This should be inside the `#if ENABLE(TOUCH_EVENTS) && !ENABLE(IOS_TOUCH_EVENTS)` a couple of lines above.

Ahmad-S792 replied:
Done. Pushed new commit with fix.

Ahmad-S792 committed:
Reset 'm_touchPressed' to false in EventHandler::clear()

Reset 'm_touchPressed' to false in EventHandler::clear()
https://bugs.webkit.org/show_bug.cgi?id=250952

Reviewed by Aditya Keerthi.

Merge - https://chromium.googlesource.com/chromium/blink/+/b7791c376d099dbb0ecebf51203af5b856afb7f6

The member variable 'm_touchPressed' in EventHandler is true if there
is at least one active touchpoint. This should be reset to false
in EventHandler::clear(), otherwise mousemove events may be
incorrectly treated.

* Source/WebCore/page/EventHandler.cpp:
(EventHandler::clear): Explicitly set 'm_touchPressed' to 'false'

Canonical link: https://commits.webkit.org/259288@main
